README.md

```diff
@@ -3,30 +3,30 @@
 
 <p align="left" >
 <img src="https://s-media-cache-ak0.pinimg.com/736x/aa/bc/3b/aabc3b2b789f478ffb87ac2f0bdd2d33--ergo-proxy-manga-anime.jpg" width="250" align="center" />
-<span>Ergo Proxy - The reverse proxy agent for local domain management.</span>
+<span>Ergo Proxy - Reverse proxy agent for local domain.</span>
 
 </p>
 
 <p align="center">
-  The management of multiple apps running over different ports made easy through custom local domains.
+  Management of multiple apps running on different ports made easy through local domains.
 </p>
 
 ## Demo
 
 <img src="https://raw.githubusercontent.com/cristianoliveira/ergo/master/demo.gif" align="center" />
 
-See more on [examples](https://github.com/cristianoliveira/ergo/tree/master/examples)
+See more [examples](https://github.com/cristianoliveira/ergo/tree/master/examples)
 
-Ergo's goal is to be a simple reverse proxy that follows the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of doing only one thing and do it well. Simplicity means no magic involved. Just a flexible reverse proxy that extends the well-known `/etc/hosts` declaration.
+Ergo's goal is to be a simple reverse proxy that follows [Unix's philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of doing only one thing and doing it well. Simplicity means no magic involved. Just a flexible reverse proxy that extends the well-known `/etc/hosts` declaration.
 
 **Feedback**
 
-This project is under development but it's ready to use. Feel free to give me
+This project is still under development but it's ready to use. Feel free to leave your 
 feedback and opening issues. Suggestions and contributions are welcome. :)
 
 ## Why?
 
-Dealing with multiple apps locally is really annoying. Having to remember each port that represents each service, and this gets even worse when you have microservices. So I wanted a simple way to give each app a proper local domain. Ergos comes to solve this simple problem.
+When dealing with multiple apps locally it's really annoying rememberinf which port represents each service and it gets even worse when you have microservices. Ergo solves this by assigning a local domain for each app.
```

Comment:
- I would stick with just `Dealing with multiple...`.
- There is a typo here `rememberinf which` :)

The rest makes sense to me. It makes more impersonal.